src/vm/ClrEtwAll.man

```diff
@@ -2796,13 +2795,6 @@
                            symbol="ThreadPoolIODequeue"
                            message="$(string.RuntimePublisher.ThreadPoolIODequeueEventMessage)"/>
 
-                    <event value="65" version="0" level="win:Verbose"  template="ThreadPoolIOWork"
```

Comment:
@vancem Is it ok to delete this event  from here?

Reply:
See my comment above.   This event should be replaced by a managed equivalent.   
Removing the definition is fine (since it is unused), but it is not a good idea to reuse dthe event ID (in this case 65),  because parsers don't know if you are old code (using old defs) or new code (using the reused ID). 

Thius it is best to replace this defintation with a commend that event 65 is an obsolete event ThreadPoolIOWork so we remember not to use the event ID for new events.

Reply:
Other defunct definitions in the file had just a comment above them, so I followed that style and kept the actual XML tags intact.